medyator/request_handler.py

```
@@ -1,6 +1,6 @@
-from typing import Generic, TypeVar
+from typing import Awaitable, Generic, TypeVar
 
-from .contracts import Command, Query
+from .contracts import AsyncCommand, AsyncQuery, Command, Query
```

Ruff (0.11.9)

`.contracts.AsyncCommand` imported but unused

Remove unused import

(F401)